zephyr/module.yml

```diff
@@ -34,3 +34,11 @@ blobs:
     license-path: tlsr9/ble/license.txt
     url: http://wiki.telink-semi.cn/wiki/protocols/Zephyr/binaries/public/tl321x_lib/lib_zephyr_tl321x_37251a7ca8f8488443f70dc6a09de19b604894a4.a
     description: "Binary libraries supporting Telink TL321X series BLE subsystems"
+
+  - path: lib_zephyr_tl321x_pm.a
+    sha256: be742f1033d1414e799bfec7882f38bb5117c0ef6fdbca14c402a45e24891f26
+    type: lib
+    version: 'V4.0.4.2'
+    license-path: tlsr9/ble/license.txt
+    url: http://wiki.telink-semi.cn/wiki/protocols/Zephyr/binaries/public/tl321x_pm_lib/lib_zephyr_tl321x_37251a7ca8f8488443f70dc6a09de19b604894a4.a
+    description: "Binary libraries supporting Telink TL321X series BLE subsystems in pm mode"
```

Comment:
Binary libraries for the Telink TL321X series BLE subsystems, optimized for low-power mode

Reply:
already done